pkgs/tools/nix/nixos-render-docs/src/nixos_render_docs/manual.py

```diff
@@ -237,6 +237,7 @@ class HTMLParameters(NamedTuple):
     scripts: Sequence[str]
     toc_depth: int
     chunk_toc_depth: int
+    section_toc_depth: int
```

Comment:
I was a bit confused by this, assuming `toc_depth` refers to the current level of nesting but it looks like it is the number of allowed nesting levels that remain. Maybe use name like `toc_max_nesting_depth` or `toc_depth_limit`.

Reply:
Yeah, comment at least would be nice.